CantoboardFramework/Keyboard/InputController.swift

```diff
@@ -58,7 +58,7 @@ struct KeyboardState: Equatable {
     }
     
     var symbolShape: SymbolShape {
-        symbolShapeOverride ?? (keyboardContextualType == .chinese ? .full : .half)
+        symbolShapeOverride ?? (!keyboardContextualType.isEnglish ? .full : .half)
```

Comment:
If the keyboard is in mixed mode, the keyboard will always show full width symbols with this change, even if the user has just typed English. Is this the best behavior?

Reply:
Nevermind, let's stick with this for now